src/sentry/static/sentry/app/views/eventsV2/table/tableModalEditColumn.tsx

```diff
@@ -207,8 +224,15 @@ function filterAggregationByField(organization: Organization, f?: Field): Aggreg
   );
 }
 
-function filterFieldByAggregation(organization: Organization, a?: Aggregation): Field[] {
+function filterFieldByAggregation(
+  organization: Organization,
+  tagKeys: null | string[],
+  a?: Aggregation
+): Field[] {
   let fieldList = Object.keys(FIELDS);
+  if (tagKeys && tagKeys.length) {
+    fieldList = fieldList.concat(tagKeys);
```

Comment:
@markstory `fieldList` will need to be de-duplicated since the intersection of the keys of `FIELDS` and `tagKeys` is non-empty.

We want to do this since the dropdown component will only highlight the first occurrence of an item if you were to hover over the 2nd or subsequent occurrence of that same item.

Reply:
Good catch. Fixed.